src/dnfile/enums.py

```diff
@@ -61,28 +62,29 @@ def __iter__(self):
                 yield name, val
 
     def __repr__(self):
-        return '\n'.join(["{:<40}{:>8}".format(n, str(v)) for n, v in self])
+        return "\n".join(["{:<40}{:>8}".format(n, str(v)) for n, v in self])
 
 
 class CorHeaderEnum(_enum.IntEnum):
-    CLR_ILONLY          = 0x00000001
-    CLR_32BITREQUIRED   = 0x00000002
-    CLR_IL_LIBRARY      = 0x00000004
-    CLR_STRONGNAMESIGNED    = 0x00000008
-    CLR_NATIVE_ENTRYPOINT   = 0x00000010
-    CLR_TRACKDEBUGDATA      = 0x00010000
-    CLR_PREFER_32BIT        = 0x00020000
+    CLR_ILONLY = 0x00000001
+    CLR_32BITREQUIRED = 0x00000002
+    CLR_IL_LIBRARY = 0x00000004
+    CLR_STRONGNAMESIGNED = 0x00000008
+    CLR_NATIVE_ENTRYPOINT = 0x00000010
+    CLR_TRACKDEBUGDATA = 0x00010000
+    CLR_PREFER_32BIT = 0x00020000
```

Comment:
I certainly appreciate how you took the time to vertically align the elements in `enum.py`. If I had my way, I'd disable black from changing this; however, this is an example of how black is opinionated and uncompromising: there's not a way to do that. So, you have to decide if its worthwhile to keep the vertical alignment, or the consistency of black.